src/api-service/__app__/onefuzzlib/notifications/ado.py

```diff
@@ -232,13 +233,12 @@ def notify_ado(
     try:
         ado = ADO(container, filename, config, report)
         ado.process()
-    except AzureDevOpsAuthenticationError as err:
-        fail_task(report, err)
-    except AzureDevOpsClientError as err:
-        fail_task(report, err)
-    except AzureDevOpsServiceError as err:
-        fail_task(report, err)
-    except AzureDevOpsClientRequestError as err:
+    except (
```

Comment:
👍

Reply:
This wasn't meant to be included in this PR.  backing out.